src/compiler/types/validate-build-package-json.ts

```diff
@@ -150,7 +150,10 @@ export const validateCollection = (
 ) => {
   if (outputTarget.collectionDir) {
     const collectionRel = join(relative(config.rootDir, outputTarget.collectionDir), COLLECTION_MANIFEST_FILE_NAME);
-    if (!buildCtx.packageJson.collection || normalizePath(buildCtx.packageJson.collection) !== collectionRel) {
+    if (
+      !buildCtx.packageJson.collection ||
+      normalizePath(buildCtx.packageJson.collection) !== normalizePath(collectionRel)
```

Comment:
Can we add a unit test for this conditional? It looks like we have some testing of this function in:
https://github.com/ionic-team/stencil/blob/2d5d98e4f028a9a7dcb8001e2defd8a13f30d7d3/src/compiler/types/tests/validate-package-json.spec.ts#L101-L106